ddtrace/tracer/remote_config.go

```diff
@@ -75,6 +76,24 @@ func (t *tags) toMap() *map[string]interface{} {
 	return &m
 }
 
+func (t *tracer) dynamicInstrumentationRCUpdate(u remoteconfig.ProductUpdate) map[string]state.ApplyStatus {
+
+	for k, v := range u {
+		log.Info("rc configuration for %s:\n\t%s\n", k, v)
+		passFullConfiguration(k, string(v))
+	}
+
+	// XXX: This function would typically respond with a status to say if the config was acknowledged
+	// but since we're extracting from BPF, we do not have a feedback loop and
+	return nil
```

Comment:
What effect does returning `nil` here have?

If it doesn't acknowledge the config, it means the agent will keep sending it to the tracer and I'm not sure if it impacts the backend.

We should check with RC to see what we should return here.

Reply:
Thats a good question. My understanding was that it would be set to an unknown state (state.ApplyStateUnknown) but parsing through the code now I don't see that. I'll look into it more closely on Monday, but returning `state.ApplyStateUnacknowledged` seems to makes sense.